docs/api-docs/apps/guide/apps-05-oauth.mdx

```diff
@@ -134,9 +134,9 @@ BigCommerce responds to the access_token request with JSON that contains a perma
 |:---------|:-----|:------------|
 | `access_token` | string | The semi-permanent security token that your app can use to make requests on behalf of the store. Save this value securely for future requests. |
 | `scope` | string | A space-separated list of the OAuth scopes this `access_token` authorizes access to. |
-| `user.id` | integer | BigCommerce’s unique identifier for the merchant or authorized user. Save this value to identify the user in future requests. |
+| `user.id` | integer | BigCommerce’s unique identifier for the store owner or authorized user. Save this value to identify the user in future requests. |
```

Comment:
@Cdestewart There was a comment on [CSS-372](https://bigcommercecloud.atlassian.net/browse/CSS-372) that there has been a change to the information returned for the Auth callback. Are there new properties? What's the new schema?

Reply:
There is a new owner field that will return the store owner. The user will return the authorized user or the store owner as you have it listed.

Reply:
{
  "access_token": "redacted",
  "scope": "store_v2_orders store_v2_transactions store_app_extensions_manage store_channel_settings store_content_checkout store_v2_content store_v2_default store_v2_information store_inventory_read_only users_basic_information",
  "user": {
    "id": 242344,
    "email": "example.one@bigcommerce.com"
  },
  "owner": {
    "id": 3878594,
    "email": "example.two@bigcommerce.com"
  },
  "context": "stores/abcd1234",
  "account_uuid": "example-account-uuid-1234"
}

Reply:
Thanks, added